### ISSUE TITLE
get_coords() method for retrieving coordinates of pieces by colour and name

### ISSUE THREAD

**Issue by ngjunsiang** [PohAnson/ChessGame#14]
## Problem

There is no abstract method for getting a list of coordinates for pieces of a specific colour and name, e.g. `white pawn`. `separate_pieces()` is supposed to help with this but is difficult to use.

## Suggested resolution

Implement `get_coords()`, a `Board` instance method for returning a list of coordinates of matching pieces.

Pieces should be retrieved separately using `get_piece()` instead of returned by this method.

```
    def get_coords(self, colour, name):
        """
        Return a list of coords where piece colour and name match.
        Returns empty list if no such piece found.
        (Meant to be used in a for loop.)
        """
        pass
```

**Comment by ngjunsiang** [comment]
Next issue to fix after this: #15

**Comment by ngjunsiang** [comment]
Not yet merged into master
